parts/k8s/kubernetescustomscript.sh

```diff
@@ -55,6 +55,21 @@ function waitForCloudInit() {
     wait_for_file 900 1 /var/log/azure/cloud-init.complete || exit $ERR_CLOUD_INIT_TIMEOUT
 }
 
+function systemctlEnableAndStart() {
```

Comment:
Moved this function up to follow the pattern that fun declarations are always before their first invocation.

Also, increased retry count, delay, and timeout enforcement to deal with observed transient systemctl errata.